### PR TITLE
adds a runtime locale to the array of locales on disk

### PR DESCRIPTION
**Pull request by kellyselden**
addresses #281

### CODE REVIEW

**Comment by jamesarosen** [comment]
Looks like we're still blocked on ember-beta and -canary :/

**Comment by jamesarosen** [comment]
Blocked by #284

**Comment by kellyselden** [comment]
rebased and passing

File: tests/unit/utils/locale-test.js [hunk 1]
```diff
@@ -4,16 +4,51 @@ moduleFor('service:i18n', 'I18nService#locales', {
...
+    'es',
+    'it'
+  ];
+
+  const i18n = this.subject({ locale: 'en' });
+
+  i18n.addTranslations('it', {});
+  i18n.addTranslations('it', {});
+
+  assert.deepEqual(i18n.get('locales'), locales);
```
Comment (jamesarosen) [inline tests/unit/utils/locale-test.js]: I don't understand how this test is passing.
```js
x = Ember.A()
x.pushObject('it')
x.pushObject('it')
x // [ 'it', 'it' ]
```

**Comment by jamesarosen** [comment]
We should probably change `get-locales` to start with an [`Ember.Set`](http://emberjs.com/api/classes/Ember.Set.html) and use `add()` instead of `pushObject()`.

File: tests/unit/utils/locale-test.js (hunk 1 above)
Comment (kellyselden) [inline tests/unit/utils/locale-test.js]: I'm using addObject, it does the duplicate ignoring for me.

**Comment by kellyselden** [comment]
@jamesarosen I'm not using `pushObject`. Also what do you mean by "start with an `Ember.set`"?

**Comment by jamesarosen** [comment]
I didn't catch the distinction between `addObject` and `pushObject`. Nice.

**Comment by kellyselden** [comment]
Thanks.

**Pull request merged**